src/bundle.rs

```diff
@@ -290,7 +290,13 @@ extra_1,  data, 0x06,            ,   20K,
 
         let image_names = zip
             .file_names()
-            .filter(|file_name| file_name.starts_with(Self::IMAGES_PREFIX))
+            .filter(|file_name| {
```

Comment:
Supernit: I have the feeling we can simplify the `filter` and the following `filter_map` roughly as follows:
```rust
.filter_map(|file_name|
    file_name.strip_prefix(Self::IMAGES_PREFIX).and_then(|file_name| (!file_name.is_empty()).then(|| file_name.to_string())))
```
(excuse any typos)

If you could maybe try this out? (I get it that monadic comprehensions are subjective, but trying to avoid the `.unwrap()` in there.)

Also, would you add the exact same filtering to the `efuses` directory, as it suffers the same problem?

Reply:
Your solution removes the prefix from the collected filenames.
I had to re-add it after, it is not very nice (and I am not sure if the `/` is cross-platform.
```rust
            .filter_map(|file_name| {
                file_name
                    .strip_prefix(Self::IMAGES_PREFIX)
                    .and_then(|file_name| {
                        (!file_name.is_empty())
                            .then(|| Self::IMAGES_PREFIX.to_string() + "/" + file_name)
                    })
```

Have you another suggestion, I am not used to functional programing yet :)

Reply:
Or simpler:
```rust
.filter(|file_name| {
    file_name.starts_with(Self::IMAGES_PREFIX) && !file_name.ends_with("/")
})
```

Reply:
Ah yes; I didn't actually check that the prefix needs to stay! Tour second suggestion should do fine I think.